readme.md

```diff
@@ -299,6 +299,7 @@
 - [Functional Programming](https://github.com/lucasviola/awesome-functional-programming#readme)
 - [Static Analysis & Code Quality](https://github.com/mre/awesome-static-analysis#readme)
 - [Information Retrieval](https://github.com/harpribot/awesome-information-retrieval#readme) - Learn to develop your own search engine.
+- [Graph Embedding](https://github.com/benedekrozemberczki/awesome-graph-embedding#readme) - Learn representations of graphs.
```

Comment:
This is not an accurate description "graph embedding". It makes it sound like your list is about learning the representations of graphs.

https://datascience.stackexchange.com/questions/24081/what-are-graph-embedding

Reply:
Yes. Because it is about learning the representations of whole graphs such way that graphs with low graph edit distance are close in the embedding space. In the linked stackexchange thread they use the term wrongly to node embedding /network representation learning, where the task is to map nodes of a single graph into a latent space such way that a certain proximity measure on the graph (e.g. shortest path distance) is approximately preserved.  Please look at these repositories.

https://github.com/thunlp/NRLPapers
https://github.com/chihming/awesome-network-embedding

Here the terminology is consistent with the one that I use.

Reply:
Alright, but the description should at least be `Learn representations of graphs.` => `Learn the representations of graphs.`, right?

Reply:
Yes. That is better.

Reply:
Can You update the commit?

Reply:
No, that's your responsibility.